public/js/index.js

```diff
@@ -38,10 +38,11 @@ formElement.addEventListener("submit", function (e) {
 	addLesson();
 });
 
-window.onload = handleClearBtn;
+
 modalLessonClose.addEventListener("click", handleCloseLessonModal);
 clearBtn.addEventListener("click", handleClear);
 overlay.addEventListener("click", handleCloseLessonModal);
+formElement.addEventListener("keyup", handleClearBtn);
```

Comment:
Changed the handleClearBtn function so it listens to the form element as you mentioned.

Reply:
Thanks – this works great now!